helm/templates/hpa-cron.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.hpa.notify-bc-cron.enabled -}}
```

Comment:
I think we need an `{{- end }}` at the bottom to close off the `{{- if...` here.